lib/board.rb

```diff
@@ -15,6 +15,12 @@ def display
     HEREDOC
   end
 
+  def available_moves
+    open_squares = []
+    @squares.each_with_index {|_value, index|  open_squares << index if !position_taken?(index)}
```

Comment:
If you don't want to use `!position_taken`, you can change your block to `open_squares << index unless position_taken?(index)`